components/docs-chef-io/content/automate/servicenow_integration_reference.md

```diff
@@ -0,0 +1,202 @@
+
+{{< figure src="/images/automate/snow_integration_transform_map_scripts.png" alt="Script Includes">}}
+
+### Scripted REST API
+
+The **Scripted Rest API** feature aids in establishing a connection between ServiceNow and the Chef Automate application with authentication. You can navigate this section by selecting **Chef Automate** > **Script Rest API** from ServiceNow.
+
+{{< figure src="/images/automate/snow_integration_scripted_restapi_1.png" alt="Scripted REST Service">}}
+
+{{< figure src="/images/automate/snow_integration_scripted_restapi_2.png" alt="Scripted REST Resource">}}
+
+## Transform Maps
+
+The **Transform Maps** feature maps the source table, asset import to the target table, and the CMDB Server. You can navigate to the **Transform Maps** section by selecting **Chef Automate** > **Transform Maps** from ServiceNow, and make the required changes.
+
+{{< figure src="/images/automate/snow_integration_field_map.png" alt="Table Transaction Map">}}
+
+### Field Maps
+
+The **Field Maps** establishes a relationship between a field in an import set table and the target table.
+
+| Source field | Target field |
+| :---        | :---   |
+| asset_tag | asset_tag |
+| attributes | attributes |
+| cpu_core_count | cpu_core_count |
+| cpu_count | cpu_count |
+| cpu_manufacturer | cpu_manufacturer |
+| cpu_speed | cpu_speed |
+| cpu_type | cpu_type |
+| disk_space | disk_space |
+| dns_domain | dns_domain |
+| host_name | host_name |
+| ip_address | ip_address |
+| mac_address | mac_address |
+| manufacturer | manufacturer |
+| model | model_id |
+| model_category_name | x_chef_automate_model_categoryname |
+| name | name |
+| os | os |
+| os_service_pack | os_service_pack |
+| os_version | os_version |
+| ram | ram |
+| serial_number | serial_number |
+| short_description | short_description |
+| system_class_name | sys_class_name |
+| virtual | virtual |
+| [Script] | install_status |
+
+### Transform Map Scripts
+
+**Transformation** events occur while transforming an import set table into a target table.
+
+`On Before`
+: Mapping manufacturer, CPU manufacturer, model category, and model by using this script.
+
+`On After`
+: The five `On After` scripts required to update the values are:
+
+  1. Update **node id**, **node name**, **organization**, **IP address** and insert the record into Chef Automate client run cookbooks, Chef Automate client run recipes, Chef Automate client run lists.
+  1. Update **software**, **name** and installed in _CMDB software instance_ table.
+  1. Update data in the _cmdb\_ci\_spkg_ table and the _cmdb\_software\_instance_ table.
+  1. Insert or update data in the _cmdb\_ci\_file\_system_ table.
+  1. Insert or update data in the _cmdb\_ci\_network\_adapter_ table.
+
+{{< figure src="/images/automate/snow_integration_transform_map_scripts1.png" alt="Transform Map Scripts">}}
+
+### Roles
```

Comment:
Roles should be a main heading and not a part of transform maps

Reply:
done